osu.Game/OsuGame.cs

```diff
@@ -1006,6 +1006,9 @@ private void showOverlayAboveOthers(OverlayContainer overlay, OverlayContainer[]
             if (overlay.IsPresent)
                 return;
 
+            Settings.Hide();
+            Notifications.Hide();
```

Comment:
This doesn't seem to fix the original issue (opening changelog via settings footer).

This is because the footer calls `changelog.ShowBuild` and schedules something, making `overlay.IsPresent` true and early returning:
https://github.com/ppy/osu/blob/49f530910c903479939654edbcdadd8138882ee3/osu.Game/Overlays/ChangelogOverlay.cs#L25

Not sure if putting above that condition is the solution,

Reply:
Good catch, have added test coverage using that method call specifically.